src/main/java/edu/wpi/cs3733/D22/teamC/DBManager.java

```diff
@@ -146,4 +148,35 @@ public void initializeLocationTable() {
                 emptyTables
         );
     }
+    /**
+    Initialize Service Request Table
+     */
+    public void initializeSRTable() {
+        initializeTable("SERVICE_REQUESTS", "CREATE TABLE SERVICE_REQUESTS (\n" +
+                                                                        "    REQUESTID char(10),\n" +
+                                                                        "    CREATORID char(10),\n" +
+                                                                        "    ASSIGNEEID char(10),\n" +
+                                                                        "    LOCATIONID char(10),\n" +
+                                                                        "    CREATIONTIMESTAMP DATETIME,\n" +
+                                                                        "    STATUS  varchar2(50),\n" +
+                                                                        "    PRIORITY   varchar2(50),\n" +
+                                                                        "    REQUESTTYPE varchar2(50),\n" +
+                                                                        "    DESCRIPTION varchar2(150),\n" +
+                                                                        "    CONSTRAINT PK_REQUESTID PRIMARY KEY (REQUESTID))\n"
+                        , emptyTables);
+
+    }
+
+    /**
+     * Initialize Medical Equipment Service Request Table
+     */
+    public void initializeMedEquipSRTable() {
+        initializeTable("MED_EQUP_SERVICE_REQUESTS", "CREATE TABLE MEDICAL_EQUIP_SERVICE_REQUESTS (\n" +
```

Comment:
1. Same questions about pluralization and **\n** that I have previously listed about SERVICE_REQUESTS applies here.
2. Two different TABLES referenced in these commands. **MED_EQUP_SERVICE_REQUESTS** is not **MEDICAL_EQUIP_SERVICE_REQUESTS**. We need to stick to 1 (I would advocate for the latter).